src/Npgsql/NpgsqlRawCopyStream.cs

```diff
@@ -543,18 +559,24 @@ internal NpgsqlCopyTextReader(NpgsqlConnector connector, NpgsqlRawCopyStream und
         }
 
         /// <summary>
-        /// Cancels and terminates an ongoing import.
+        /// Cancels and terminates an ongoing export.
```

Comment:
Is this correct?

Reply:
Well, it is `NpgsqlCopyTextReader`, which we're supposed to use while exporting from the database.